test/unit/device/rownt_layout.hpp

```diff
@@ -51,8 +51,10 @@ namespace rocwmma
             BlockDim = BlockN,
             KDim     = BlockM,
 
-            MaxVectorWidth = detail::VecWidthTraits<BlockDim, KDim, DataT>::MaxVectorWidth,
-            VectorWidth    = std::is_same<LayoutP, row_major>::value ? MaxVectorWidth : 1
+            MaxVectorWidth = std::is_same<DataT, float64_t>::value
+                                    ? 1
+                                    : 4, // Actual output of the mfma hardware
```

Comment:
Instead of 4, can you use detail::VecWidthTraits<BlockDim, KDim, DataT>::MaxVectorWidth ?

This test is not exclusive to accumulator frags but B as well.